hibernate-validator/src/main/java/io/micronaut/configuration/hibernate/validator/HibernateMessageBundle.java

```diff
@@ -0,0 +1,19 @@
+package io.micronaut.configuration.hibernate.validator;
+
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.context.i18n.ResourceBundleMessageSource;
+import io.micronaut.core.order.Ordered;
+import jakarta.inject.Singleton;
+
+@Singleton
+@Requires(bean = MicronautHibernateValidator.class)
```

Comment:
Would it be better to use this?
```suggestion
@Requires(resources = "classpath:org/hibernate/validator/ValidationMessages.properties")
```